sale_measure/measure.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<openerp>
+    <data>
+
+<!-- ===================== This is free layout for mesure.mesure =============================-->
+	  <record model="ir.ui.view" id="view_measure_tree">
+            <field name="model">measure.measure</field>
+	    <field name="type">tree</field>
+            <field name="arch" type="xml"> 
+		<tree string="Measure">
+		    <field name="product_id" string = "Produit"/> 
```

Comment:
Do not overwritre the name here. If you want to translate it, use translation file